ui/src/app/machines/views/MachineDetails/MachineSummary/SystemCard/SystemCard.tsx

```diff
@@ -80,11 +80,13 @@ const SystemCard = ({ id }: Props): JSX.Element => {
   }
 
   return (
-    <Card className="machine-summary__system-card">
-      <strong className="p-muted-heading u-sv1">System</strong>
-      <hr />
-      {content}
-    </Card>
+    <div className="machine-summary__system-card">
```

Comment:
This is to separate the CSS grid space from the actual card itself, so that the card only takes up as much height as it needs to. Without this change the system card could stretch a lot, depending on how much content is in the network and workload annotations cards.